src/modules.js

```diff
@@ -458,7 +458,8 @@ var LibraryManager = {
       'library_glew.js',
       'library_html5.js',
       'library_signals.js',
-      'library_async.js'
+      'library_async.js',
+      'library_vr.js'
```

Comment:
Well, let's do it not in this pull itself, seems unfair to delay it on such a discussion.